dotnet-interactive/HttpApiBootstrapper.cs

```diff
@@ -12,34 +12,50 @@ public static string GetJSCode(Uri apiRoot)
         {
             var id = $"{Process.GetCurrentProcess().Id}.{apiRoot.Port}";
             var template = @"#!javascript
-if ((typeof(requirejs) !==  typeof(Function)) || (typeof(requirejs.config) !== typeof(Function))) { 
-    let script = document.createElement(""script""); 
-    script.setAttribute(""src"", ""https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js""); 
-    script.onload = function(){
+// ensure `requirejs` is available
+if ((typeof(requirejs) !==  typeof(Function)) || (typeof(requirejs.config) !== typeof(Function))) {
+    let script = document.createElement('script');
+    script.setAttribute('src', 'https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js');
+    script.onload = function() {
         loadDotnetInteractiveApi();
     };
-    document.getElementsByTagName(""head"")[0].appendChild(script); 
+
+    document.getElementsByTagName('head')[0].appendChild(script);
 }
 else {
     loadDotnetInteractiveApi();
 }
 
-function loadDotnetInteractiveApi(){
-    let apiRequire = requirejs.config({context:""dotnet-interactive.$SEED$"",paths:{dotnetInteractive:""$API_URL$""}});
-    apiRequire(['dotnetInteractive'], 
-    function(api) {       
-        api.createDotnetInteractiveClient(""$HOST$"", window);
-    },
-    function(error){
-        console.log(error);
-    });
+function loadDotnetInteractiveApi() {
+    // ensure interactive helpers are loaded
+    let interactiveHelperElement = document.getElementById('dotnet-interactive-script-loaded');
```

Comment:
This sentinel element is just so that we don't create the global function `createDotnetInteractiveClient` repeatedly.